Gemfile

```diff
@@ -12,7 +12,8 @@ group :test do
   gem "chefstyle", "1.7.5"
   gem "rake"
   gem "rspec", "~> 3.0"
-  gem "parallel", "~> 1.20.1" #pin parallel to 1.20.1 until we support ruby 2.4
+  gem "parallel", "~> 1.20.1"   # pin parallel to 1.20.1 until we support ruby 2.4
```

Comment:
We can probably just drop ruby 2.4 support in the tests/gemspec now

Reply:
Sure @tas50,thanks.

Reply:
These pins can go away now